portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +31,55 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     public ArrayList<String> list = new ArrayList<String>();
+    ArrayList<String> comments = new ArrayList<String>();
```

Comment:
I don't think you need this anymore since you're returning things directly from DataStore
```suggestion
```